modules/product/src/main/java/com/opengamma/strata/product/fx/FxSingle.java

```diff
@@ -138,9 +138,8 @@ public static FxSingle of(
   /**
    * Creates an {@code FxSingle} using a rate.
    * <p>
-   * This create an FX specifying a value date, notional in one currency, the second currency
-   * and the FX rate between the two.
-   * The currencies of the payments must differ.
+   * This create an FX specifying the amount, FX rate and value date.
```

Comment:
This sentence doesn't quite make sense. I suspect it was intended to be "This creates an `FxSingle` ...".